packages/iota/src/clients/singleNodeClient.ts

```diff
@@ -551,7 +596,7 @@ export class SingleNodeClient implements IClient {
         const response = await this.fetchWithTimeout(
             method,
             `${basePath}${route}`,
-            { "Content-Type": "application/octet-stream" },
+            { "Content-Type": "application/vnd.iota.serializer-v1" },
```

Comment:
```suggestion
            { "Accept": "application/vnd.iota.serializer-v1" },
```
The response content type is controlled through passing in tyhe accept header, see how it is implemented in hornet: https://github.com/gohornet/hornet/blob/307017aafcfa1c3c13490114c3476b6ddcd17025/pkg/restapi/restapi.go#L100